daemon/src/actors.rs

```diff
@@ -25,6 +25,7 @@ impl Neovim {
     /// Will panic if Neovim cannot be started or if the file cannot be opened.
     pub async fn new(file_path: PathBuf) -> Self {
         let handler = Dummy::new();
+        /* TODO: consider a process to bootstrap nvim for testing in CI. */
```

Comment:
I also didn't notice that your github actions workflow installs it from a tarball!! I first actually left a TODO mentioning [`spack-rs`](https://docs.rs/spack-rs) ([repo](https://github.com/cosmicexplorer/spack-rs)), my cargo wrapper for [`spack`](https://github.com/spack/spack) which can bootstrap anything spack can, including neovim. However, there's some [optimization for bootstrapping](https://github.com/spack/spack/discussions/44714) I have yet to do, and `spack-rs` in particular is for linking libraries--no reason to use it  over `spack` itself for executables.

spack is a really interesting and useful project (build any codebase on any platform without any installation requirements, install multiple versions of each package, [really cool dependency "spec" syntax](https://spack.readthedocs.io/en/latest/basic_usage.html#specs-dependencies)). You can think of it as similar to nix but with a dependency resolver and no install requirements. But since I realized we're not linking any libraries and nvim is rather easy to install, I can't immediately see where spack would be useful for ethersync at this moment.

I have removed this comment, because I didn't realize CI already bootstraps nvim. If `ethersync` ever exports a library instead of just a binary, or it wants to depend on any C/C++ library, then it may become relevant.

Please feel free to ignore this comment, I like spack quite a lot but I think it's not useful right now. ^_^